image-loader/test/scala/lib/ImageProcessorLoaderTest.scala

```diff
@@ -65,27 +68,20 @@ class ImageProcessorLoaderTest extends FreeSpec with Matchers with EitherValues
       val instance = ImageProcessorLoader.loadImageProcessor("scala.lib.ImageProcessorThatDoesntExist", testConfig)
       instance.left.value shouldBe "Unable to find image processor class scala.lib.ImageProcessorThatDoesntExist"
     }
-
-    "should fail to load a no arg processor that doesn't take configuration with non-empty configuration" in {
-      val instance = ImageProcessorLoader.loadImageProcessor(classOf[NoArgImageProcessor].getCanonicalName, testNonEmptyConfig)
-      instance.left.value shouldBe "Attempt to initialise image processor scala.lib.NoArgImageProcessor failed as configuration is provided but the constructor does not take configuration as an argument."
-    }
-
-    "should fail to load an object processor that doesn't take configuration with non-empty configuration" in {
-      val instance = ImageProcessorLoader.loadImageProcessor(ObjectImageProcessor.getClass.getCanonicalName, testNonEmptyConfig)
-      instance.left.value shouldBe "Attempt to initialise image processor scala.lib.ObjectImageProcessor$ failed as configuration is provided but the constructor does not take configuration as an argument."
-    }
```

Comment:
I think both of these tests are useful to have and can be retained more easily be introducing a resources type.